docs/connectors/building-connectors/connector-configuration.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Connector Configuration Files"
+sidebar_position: 2
+---
+
+There special files that should be implemented to describe your connector.
+
+### `spec.go`
+
+This file defines the public-facing characteristics of the connector such as its name and capabilities. Begin by updating placeholder values with the specific details of your connector.
+
+```go
+var version = "v0.1.0"
+
+// Specification returns the connector's specification.
+func Specification() sdk.Specification {
+ return sdk.Specification{
+  Name:        "file-sync",
+  Summary:     "<describe your connector>",
+  Description: "<describe your connector in detail>",
+  Version:     version,
+  Author:      "<your name>",
+ }
+}
+```
+
+### `config.go`
```

Comment:
this is more of a suggested approach for the developer to use, not a requirement, could be any file name, could also be just a struct in the `source.go` file.

Reply:
with this said, maybe the whole page should be renamed too

Reply:
maybe just get rid if the whole config file thing here, name the file "Connector Specifications" and just talk about the specification function and how it's needed for both the source and destination

Reply:
Just made the change here: [e8a32c1](https://github.com/ConduitIO/conduit-site/pull/80/commits/e8a32c1991d01380cb956734349e921af546f1a1)